manim/utils/config_utils.py

```diff
@@ -373,24 +373,24 @@ def _run_config():
     config_files = _paths_config_file()
     if _from_command_line():
         args = _parse_cli(sys.argv[1:])
-        if args.config_file is not None:
-            if os.path.exists(args.config_file):
-                config_files.append(args.config_file)
-            else:
-                raise FileNotFoundError(f"Config file {args.config_file} doesn't exist")
-        else:
-            script_directory_file_config = os.path.join(
-                os.path.dirname(args.file), "manim.cfg"
-            )
-            if os.path.exists(script_directory_file_config):
-                config_files.append(script_directory_file_config)
-
     else:
         # In this case, we still need an empty args object.
         args = _parse_cli([], input=False)
         # Need to populate the options left out
         args.file, args.scene_names, args.output_file = "", "", ""
 
+    if args.config_file is not None:
```

Comment:
**Question** (last one, I swear) : I think that this will never be satisfied, as every args are set to `None` by l.378 (except `file`, `scene_names `and `output_file` (l.380). If I'm correct, we should remove this.

Reply:
I disagree this condition is tested on both the case when manim is run from the command line or from another script.
This condition is indeed never satisfied when run from the command line but it can be satisfied on the other case.
We could place the l.382 from l.386 into the `if _from_command_line()` but then lines l.388 to l.392 would have to appear twice (either directly copy/pasted or from a function) one for the case run from command line and another when run from a script.

Reply:
Oops. My stupidity strikes again. It was obvious, I don't know what came across my mind why I wrote that, sorry lol.